google-cloud-translate/test/google/cloud/translate/v2/service_test.rb

```diff
@@ -14,9 +14,19 @@
 
 require "helper"
 
+class FakeClient
```

Comment:
would be nice to comment that this FakeClient is tailored for the Service.sign_http_request! method

Reply:
done